Source/WebKit/GPUProcess/webrtc/LibWebRTCCodecsProxy.mm

```diff
@@ -240,10 +240,10 @@ static bool validateCodecString(WebCore::VideoCodecType codecType, const String&
 
 void LibWebRTCCodecsProxy::flushDecoder(VideoDecoderIdentifier identifier, CompletionHandler<void()>&& completionHandler)
 {
-    doDecoderTask(identifier, [&](auto& decoder) {
+    doDecoderTask(identifier, [protectedThis = Ref { *this }, &completionHandler](auto& decoder) {
```

Comment:
It should not be needed to do protectedThis since the callback is executed synchronously.
Would doDecoderTask taking a const Function<>& solve the warning?